### PR TITLE
Add prettier config

### LINKED ISSUE #98: ✨❓ Feature Request: Add Prettier config
@MidouWebDev 
**Description**
Add prettier config file. So everyone working will keep the same style. 

Optional - Can also add a husky pre-commit, to check if the style is correct on commit

### PR DESCRIPTION
**Pull request by Kevan-Y**
# Pull Request

## Description

Add Prettier config.
## Related Issue

Closes #98 

## Scope

Did you remember to update the `package.json` version number?

- [ ] Major Version Update: X.\_.\_ (for complete code refactors, or large PRs that adjust most of the codebase)
- [ ] Minor Version Update: \_.X.\_ (for the addition of new commands)
- [ ] Patch Version Update: \_.\_.X (for bug fixes _in the code_.)
- [ ] No Version Update: \_.\_.\_ (no version update for additions to tests, documentation, or anything that isn't end-user facing.)

## Documentation

For _any_ version updates, please verify if the documentation page needs an update. If it does, please create an issue to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

**Comment by Kevan-Y** [comment]
@j-dogcoder @MidouWebDev  Let's me know if this config of prettier is okay for you guys. 
If so I will have another commit to apply this change into all the files

**Review by MidouWebDev — APPROVED** [review]
It is okey for me !

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "embeddedLanguageFormatting": "auto",
+  "endOfLine": "lf",
+  "htmlWhitespaceSensitivity": "css",
+  "insertPragma": false,
+  "jsxSingleQuote": false,
+  "proseWrap": "preserve",
+  "quoteProps": "as-needed",
+  "requirePragma": false,
+  "singleQuote": true,
+  "tabWidth": 2,
```
Comment (Panquesito7) [inline .prettierrc.json:13]: Shouldn't we use `4` instead?

Comment (Kevan-Y) [reply .prettierrc.json:13]: 2 is my own preference, 4 can be a bit more space for clarity.

Comment (Panquesito7) [reply .prettierrc.json:13]: > 2 is my own preference, 4 can be a bit more space for clarity.

Let's see what the other maintainers think. 🙂

Comment (j-dogcoder) [reply .prettierrc.json:13]: Let’s go with 4

**Comment by j-dogcoder** [comment]
Is this ready for review @Kevan-Y?

**Comment by Kevan-Y** [comment]
> Is this ready for review @Kevan-Y?

Sorry for the delay. Yes it's ready to review

**Comment by Kevan-Y** [comment]
The last commit is to apply the prettier change to all the affected files. 
Technically it should not break anything. 

@j-dogcoder , @MidouWebDev , @Panquesito7 
Should I remove prettier for MD files? It is changing some upper case to lower case

**Review by j-dogcoder — APPROVED** [review]

**Comment by Panquesito7** [comment]
> The last commit is to apply the prettier change to all the affected files. Technically it should not break anything.
> 
> @j-dogcoder , @MidouWebDev , @Panquesito7 Should I remove prettier for MD files? It is changing some upper case to lower case

Yeah, I think you should do that. Feel free to create another PR for that. 🙂

**Review by Panquesito7 — COMMENTED** [review]
LGTM! 🚀

**Pull request merged**